src/components/Forms/AddServerForm.vue

```diff
@@ -76,6 +76,14 @@ export default Vue.extend({
     async connectToServer(): Promise<void> {
       this.loading = true;
 
+      if (!this.serverUrl.match(/^https?:\/\//)) {
+        this.serverUrl = 'http://' + this.serverUrl;
+      }
+
+      if (!this.serverUrl.match(/:[0-9]+$/)) {
+        this.serverUrl = this.serverUrl + ':8096';
+      }
```

Comment:
Any reason to add the rules here instead of doing it in vee-validate?

Reply:
No. I will move them